docs/source/sysadmin_guide/setting_allowed_txns.rst

```diff
@@ -0,0 +1,79 @@
+************************************************
+Setting the Allowed Transaction Types (Optional)
+************************************************
+
+By default, a validator accepts transactions from any transaction processor.
+However, Sawtooth allows you to limit the types of transactions that can be
+submitted. This on-chain setting increases the Sawtooth network's security by
+ignoring any unrecognized transaction processors.
+
+This configuration change is done once, usually on the same node that created
```

Comment:
This change can happen many times, and not just in genesis.

Reply:
Agree with @aludvik that this is not correct. It is often set in the genesis block initially, but can also be changed at anytime by submitting a transaction with the new value for the setting.  These changes maybe made as transaction processors are added to a network or there are version changes in the existing transaction processors.

Reply:
I'll fix the wording.  I was trying to explain that this is an on-chain setting so only one node needs to run this command, but I got all the words wrong. I'll push a fix soon with an improvement.